### ISSUE TITLE
UPDATE (Works with one dash '-') Bundle-Migration:  A positional parameter cannot be found that accepts argument '--force'

### ISSUE THREAD

**Issue by JeepNL** [dotnet/efcore#31218]
- Visual Studio Preview Latest (Version 17.7.0 Preview 2.0)
 - .NET & EF CORE 8 Preview 5

When I run the `Bundle-Migration` command in Visual Studio Package Manager Console with `--force`  I'm getting the error below

**Command:**
```
PM> Bundle-Migration -SelfContained -TargetRuntime linux-x64 --force
```

**Error:**
```
Bundle-Migration : A positional parameter cannot be found that accepts argument '--force'.
At line:1 char:1
+ Bundle-Migration -SelfContained -TargetRuntime linux-x64 --force
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Bundle-Migration], ParameterBindingException
    + FullyQualifiedErrorId : PositionalParameterNotFound,Bundle-Migration
```

**Comment by ErikEJ** [comment]
Are you looking at the correct docs??

**Comment by JeepNL** [comment]
Why would you think I'm looking at the wrong docs? 

And the question without a link to the 'right' docs doesn't add much value IMHO.

**Comment by ErikEJ** [comment]
https://learn.microsoft.com/en-us/ef/core/cli/powershell#bundle-migration

**Comment by JeepNL** [comment]
Thank you! 

https://learn.microsoft.com/en-us/ef/core/managing-schemas/migrations/applying?tabs=dotnet-core-cli#migration-bundle-example 

![Screenshot 2023-07-10 225543](https://github.com/dotnet/efcore/assets/39458532/153f677f-8ab8-4fb8-bbd1-aa3098c9b871)

Guess I should reopen this issue, 'cause other 'correct' docs give the wrong example.

**Comment by ajcvickers** [comment]
@JeepNL It's `--force` when using the Unix-like `dotnet ef` commands, it's `-Force` when using the PowerShell commands. (To be idiomatic to the platform.)

**Comment by JeepNL** [comment]
Okay, I didn't know that. But it was the only (-)-force example I saw in that page, so I started with double dashes. 

Maybe it's an idea to add a `-force`  example to 'Bundles' [Visual Studio] here: https://learn.microsoft.com/en-us/ef/core/managing-schemas/migrations/applying?tabs=vs#bundles 

It was a bit confusing, but that's maybe my fault.

**Comment by JeepNL** [comment]
It is confusing, because a single dash and double dashes are used together here "Bundles" [.NET Core CLI]: https://learn.microsoft.com/en-us/ef/core/managing-schemas/migrations/applying?tabs=dotnet-core-cli#bundles

`dotnet ef migrations bundle --self-contained -r linux-x64` 

And at `efbundle` [options] there's no mention of '(-)-force' : https://learn.microsoft.com/en-us/ef/core/managing-schemas/migrations/applying?tabs=dotnet-core-cli#efbundle

![Screenshot 2023-07-10 231911](https://github.com/dotnet/efcore/assets/39458532/af193327-7339-4c4b-ae11-2e94c1e1c2ec)

B.T.W. migration Bundles are *awesome* 🚀. I develop on Windows 11 w. Visual Studio but my project runs on a co-located Ubuntu 22.05 VPS and deploying DB changes is now a breeze!.

**Comment by JeepNL** [comment]
Oops! (Number 2) 'efbunle' is the executable itself. My bad.

**Comment by JeepNL** [comment]
Yes, sure. 

I just wanted to point out that in [docs](https://learn.microsoft.com/en-us/ef/core/managing-schemas/migrations/applying?tabs=vs#bundles) about Migration Bundles the `-force` option isn't mentioned but `--force` is, which is clearly visible (frame with green background)

Windows is my main O/S. I'm only using Linux for publishing and running my web projects on my VPS. It's totally unclear to me when to use a single dash or double dashes. I know there's a difference between the O/Ss, but both use single and double dashes often. 

`dotnet --info` works on both Windows and Linux. 

So, when I couldn't get it to work with `--force` I created this issue. And just after that I searched for "_A positional parameter cannot be found_" in closed issues in this repo and I found this: https://github.com/dotnet/efcore/issues/22390

After that I tried it with one dash and it worked, that's why I updated and closed this issue so soon after I created it. I felt stupid not to search in the closed issues and I don't want to waste your time, because you all are busy enough without me asking stupid questions / opening stupid issues.

I'm always a bit hesitant to create an issue, because I always think it's me who is doing something wrong. But I do think the docs would be better if there's an example with the `-force` argument.

There's another issue I encountered with Migration Bundles on Linux and that is when I publish it *without* `-SelfContained` and I try to run it on Ubuntu it complains that there's no framework. But .NET (SDK) is installed and working, I can compile the source code of my Hosted Blazor WASM / YARP / Web API projects on the VPS itself. A self contained efbundle executable is +/- 115MB and if it's not it's +/- 30MB, which  uploads faster. But this could be because I'm doing something wrong, I'm good at that, doing things wrong. I don't want to waste your time and this already costed more than enough of your time.
